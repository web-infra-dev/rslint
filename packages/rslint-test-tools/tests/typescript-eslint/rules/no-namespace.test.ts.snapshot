exports[`no-namespace > invalid 1`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-namespace",
      "messageId": "moduleSyntaxIsPreferred",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "filePath": "src/consistent-type-exports.ts",
      "range": {
        "start": {
          "line": 9,
          "column": 32
        },
        "end": {
          "line": 15,
          "column": 2
        }
      }
    },
    {
      "ruleName": "no-namespace",
      "messageId": "moduleSyntaxIsPreferred",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "filePath": "src/consistent-type-exports/type-only-exports.ts",
      "range": {
        "start": {
          "line": 6,
          "column": 28
        },
        "end": {
          "line": 14,
          "column": 2
        }
      }
    },
    {
      "ruleName": "no-namespace",
      "messageId": "moduleSyntaxIsPreferred",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 1
        },
        "end": {
          "line": 1,
          "column": 14
        }
      }
    }
  ],
  "errorCount": 3,
  "fileCount": 1,
  "ruleCount": 1
}
`;
