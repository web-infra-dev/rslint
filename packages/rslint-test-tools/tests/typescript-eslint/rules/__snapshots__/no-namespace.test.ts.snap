// Rstest Snapshot v1

exports[`no-namespace > invalid 1`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 14,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 2`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 17,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 3`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 14,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 4`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 17,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 5`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 14,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 6`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 17,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 7`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 22,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 8`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 25,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 9`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 22,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 10`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 25,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 11`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 17,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 12`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 14,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 13`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 22,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 14`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 25,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 15`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 21,
          "line": 1,
        },
        "start": {
          "column": 1,
          "line": 1,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 16`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 4,
          "line": 5,
        },
        "start": {
          "column": 3,
          "line": 3,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 2,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 17`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 4,
          "line": 5,
        },
        "start": {
          "column": 3,
          "line": 3,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 2,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 18`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 4,
          "line": 5,
        },
        "start": {
          "column": 3,
          "line": 3,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 2,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 19`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 20`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 21`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 22`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 23`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 1,
  "fileCount": 1,
  "ruleCount": 1,
}
`;

exports[`no-namespace > invalid 24`] = `
{
  "diagnostics": [
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 2,
          "line": 6,
        },
        "start": {
          "column": 1,
          "line": 2,
        },
      },
      "ruleName": "no-namespace",
    },
    {
      "filePath": "src/virtual.ts",
      "message": "ES2015 module syntax is preferred over namespaces.",
      "messageId": "moduleSyntaxIsPreferred",
      "range": {
        "end": {
          "column": 4,
          "line": 5,
        },
        "start": {
          "column": 3,
          "line": 3,
        },
      },
      "ruleName": "no-namespace",
    },
  ],
  "errorCount": 2,
  "fileCount": 1,
  "ruleCount": 1,
}
`;
